### PR TITLE
ci(cross build): splitting cross builds and upload assets

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -420,95 +412,34 @@ jobs:
...
-    - name: get_version
-      run: |
-        echo 'version<<EOF' >> $GITHUB_ENV
-        echo ${{ github.ref }} | sed -r  "s ^refs/heads/|^refs/tags/(.*) \1 g" >> $GITHUB_ENV
-        echo 'EOF' >> $GITHUB_ENV
     - uses: actions/download-artifact@v2
       with:
         name: ${{ matrix.profile }}
-        path: ./_packages/${{ matrix.profile }}
+        path: packages/${{ matrix.profile }}
```
Comment (zmstone) [inline .github/workflows/build_packages.yaml:419]: why this change ?

Comment (Rory-Z) [reply .github/workflows/build_packages.yaml:419]: No special reason, `_package` was used before because it matched the project path, but now this job doesn't need to `actions/checkout`

**Review by zmstone — APPROVED** [review]
